src/policy/fees.cpp

```diff
@@ -338,13 +325,13 @@ double TxConfirmStats::EstimateMedianVal(int confTarget, double sufficientTxVal,
         failBucket.leftMempool = failNum;
     }
 
-    LogPrint(BCLog::ESTIMATEFEE, "FeeEst: %d %s%.0f%% decay %.5f: feerate: %g from (%g - %g) %.2f%% %.1f/(%.1f %d mem %.1f out) Fail: (%g - %g) %.2f%% %.1f/(%.1f %d mem %.1f out)\n",
-             confTarget, requireGreater ? ">" : "<", 100.0 * successBreakPoint, decay,
+    LogPrint(BCLog::ESTIMATEFEE, "FeeEst: %d > %.0f%% decay %.5f: feerate: %g from (%g - %g) %.2f%% %.1f/(%.1f %d mem %.1f out) Fail: (%g - %g) %.2f%% %.1f/(%.1f %d mem %.1f out)\n",
+             confTarget, 100.0 * successBreakPoint, decay,
              median, passBucket.start, passBucket.end,
-             100 * passBucket.withinTarget / (passBucket.totalConfirmed + passBucket.inMempool + passBucket.leftMempool),
+             100 * passBucket.withinTarget / (passBucket.totalConfirmed + passBucket.inMempool + passBucket.leftMempool + std::numeric_limits<double>::min()),
```

Comment:
I'm not sure if this is the correct fix. It's very difficult to parse what this log line is doing, and adding `std::numeric_limits<double>::min()` makes it harder still. How about taking these calculations out of the log line, putting them in temporary variables, and explicitly setting them to zero if `passBucket.totalConfirmed + passBucket.inMempool + passBucket.leftMempool` is equal to zero?

Reply:
> I'm not sure if this is the correct fix

It's correct in the sense that it's not a division by 0 anymore :). Cleaned up by using temporary variables.